scripts/deploy_local.sh

```diff
@@ -163,13 +162,25 @@ check_dependencies_version() {
     err "GraphScope requires python 3.6 or greater."
     exit 1
   fi
-
+  # cmake
+  if ! command -v cmake &> /dev/null; then
+    PACKAGES_TO_UPDATE = "${PACKAGES_TO_UPDATE} cmake"
+  else
+    ver=$(cmake --version 2>&1 | awk -F ' ' '/version/ {print $3}')
+    if [[ "${ver}" < "3.1" ]]; then
+      PACKAGES_TO_UPDATE = "${PACKAGES_TO_UPDATE} cmake"
```

Comment:
In this line, the cmake should assigned a version? otherwise it doesn't work since you set 
`HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1`